### ISSUE TITLE
Xposed module not loaded

### ISSUE THREAD

**Issue by Shapirox** [Android1500/GpsSetter#121]
Keep getting the above error when launching the app. Have confirmed that module is installed and enabled.

**Comment by Android1500** [comment]
> Keep getting the above error when launching the app. Have confirmed that module is installed and enabled.

When you start to face this issue?

**Comment by Shapirox** [comment]
Yesterday, 11 days after installing. Worked fine up until then.

**Comment by hrdc69** [comment]
I'm having the same problem. i keep getting error when launching the app. The module is installed and enabled. The app still works fine and loads on my Google Pixel 8 Pro phone but not on my new Google Pixel 9 Pro XL phone.

**Comment by lnaphade** [comment]
Yes confimring not working in pixel 4XL android 13, some android system update has killed the loader

**Comment by snackchan** [comment]
Confirmed this error after I had to use https://github.com/mywalkb/LSPosed_mod on my phone to revive the LSPosed. I suppose it was caused by the August security update from Google.

**Comment by Lnlythinkrzz** [comment]
I tried all, but it still not working please help.. 
  '' 
![Uploading Screenshot_2024-10-04-21-26-29-244_com.android1500.gpssetter.jpg…]()
Enable This Module In LSPOSED Manager ''
